thrift/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -689,9 +693,10 @@ private static HttpData encodeSuccess(ServiceRequestContext ctx,
                                           SerializationFormat serializationFormat,
                                           String methodName, int seqId,
                                           TBase<?, ?> result) {
-
-        final TMemoryBuffer buf = new TMemoryBuffer(128);
-        final TProtocol outProto = ThriftProtocolFactories.get(serializationFormat).getProtocol(buf);
+        final ByteBufAllocator alloc = ctx.alloc();
+        ByteBuf buf = alloc.buffer(128);
```

Comment:
Switched to ioBuffer - just for my reference, my impression was that a context's alloc would be one that preferred direct buffers even with the normal ```buffer()``` method, and thought it might be better to trust the alloc's decision. Are there any pointers on when to use ```buffer``` vs ```ioBuffer```?